packages/api/test/server-test.js

```diff
@@ -37,16 +46,47 @@ describe('flaky express server', () => {
     });
   });
 
-  describe('/auth', async () => {
-    it('redirects to a Github url', async () => {
-      const resp = await fetch('http://0.0.0.0:3000/api/auth', {
-        headers: { redirect: 'manual' }
+  describe('delete /repo', async () => {
+    it('redirects to Github', async () => {
+      const resp = await fetch('http://0.0.0.0:3000/api/repo/fakeorg/fakerepo/test?name=dummytest', {
+        headers: { redirect: 'manual' },
+        method: 'post'
       });
-      assert.strictEqual(resp.url.includes('github.com/login'), true);
+
+      assert.deepStrictEqual(resp.url.includes('github.com/login'), true);
     });
   });
```

Comment:
Mind adding a test for generating the redirect too? something like this:

```js
    it.only('generates a GitHub redirect', async () => { 
      stubs.push(sinon.stub(repo, 'storeTicket').returns(true));
      const resp = await fetch('http://0.0.0.0:3000/api/repo/my-org/my-repo/test', {
        redirect: 'manual',
        method: 'DELETE'
      });
      assert(resp.headers.get('location').includes('github.com/login/oauth'));
    })
```

You could also make this test validate that the appropriate data from the request is being stored in the ticket (this would be a good addition to the test).